src/main/java/org/jasig/portlet/newsreader/adapter/RomeAdapter.java

```diff
@@ -31,8 +31,8 @@
 import net.sf.ehcache.Element;
 import org.apache.commons.io.IOUtils;
 import org.apache.commons.lang.StringUtils;
-import org.apache.commons.logging.Log;
-import org.apache.commons.logging.LogFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
This is great, though FWIW when you switch a project to SLF4J we rarely replace all the instances of the loggers.  We tend to do them in a class when we get to them.  Still its a welcome improvement.  Did you do this with an automated tool?  Do the automated tools handle converting all the log invocations to use the {} format?  If so that would be a reason to consider switching all log usages.